cloud_integrations_add_kubernetes.md

```diff
@@ -0,0 +1,7 @@
+### Adding a Kubernetes Cluster
+
+Adding a Kubernetes cluster for monitoring requires deploying a **Wavefront proxy** and **Wavefront collector** inside your Kubernetes cluster.
+
+You can follow the step-by-step guided Helm instructions if applicable, or follow the manual instructions to deploy the required components yourself.
+
+If you need more help, check out the [Kubernetes Troubleshooting](https://docs.wavefront.com/wf_kubernetes_troubleshooting.html) page.
```

Comment:
A lot of the Help panels follow the pattern: 

`### More Info`
<links> 
And, you could link to the Troubleshooting page and to the SDK README.